### PR TITLE
Odst 92 admins cc sign in page

### PR DESCRIPTION
**Pull request by pmeskridge**
Login page completed, but has no current functionality.

### CODE REVIEW

**Comment by yurisim** [comment]
image

On figma it looks like these are suppose to be stretched. The intent is so that, given that this is accessible on mobile, buttons need to be bigger.

image
 These are not aligned.
 
 The "Sign In" stuff is centered.
 
 
image
I want to login button isn't working

**Comment by yurisim** [comment]
SHould look like

image

**Comment by yurisim** [comment]
image

Some elements are left aligned at randomly.

**Review by ericcater — APPROVED** [review]

File: apps/ods-api/src/question/question.resolver.ts [hunk 1]
```diff
@@ -58,8 +58,8 @@ export class QuestionResolver {
       questionUpdateInput
     );
   }
-
-  @Mutation(() => QuestionGQL, { name: 'deleteQuestion' })
+  
+  @Mutation(() => QuestionGQL, { name: 'removeQuestion' })
```
[the remark this answers is not in the document]
Comment (pmeskridge) [reply apps/ods-api/src/question/question.resolver.ts:62]: Code change made and pushed.

**Pull request merged**